app/assets/javascripts/react/components/TouchKiosk.js

```diff
@@ -16,6 +16,7 @@ const mapStateToProps = (state) => {
     classroom_schedule: state.touch.classroom_schedule,
     is_fetching_slides: state.touch.is_fetching_slides,
     is_fetching_hours: state.touch.is_fetching_hours,
+    show_nav: true
```

Comment:
You want to set the default values for state the related reducer, and then pull the current value here as state.touch.show_nav.